matchbook-sdk-rest/src/main/java/com/matchbook/sdk/rest/dtos/offers/OffersGetRequest.java

```diff
@@ -144,6 +149,11 @@ public String toString() {
 
         public Init() {
             includeEdits = false;
+            sportIds = new HashSet<>();
```

Comment:
Can you please replace this code with null check, as in `PositionsRequest`, and you can make a check in getParameters like that

```java 
if (Objects.nonNull(eventIds) && !eventIds.isEmpty()) { 
... 

}
```

Reply:
I can fix those if you want before merging this PR.